records_mover/records/sources/table.py

```diff
@@ -53,6 +53,13 @@ def can_move_to_this_format(self,
             return False
         return unloader.can_unload_this_format(target_records_format)
 
+    # TODO: above can_move_to_this_format vs can_move_to_scheme - do I need to modify other PR?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker